_posts/2018-01-25-fair.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: post
+title: OME FAIR
+categories: community
+---
+
+Recently there have been several publications and substantial discussion about
+the FAIR principles (see for example,
+[Wilkinson et al, 2016](https://www.nature.com/articles/sdata201618) and the
+[Force11 Fair Data Principles](https://www.force11.org/group/fairgroup/fairprinciples)).
+Overall, the goal of the FAIR principles are “to facilitate knowledge
+discovery by assisting humans and machines in their discovery of, access to,
+integration and analysis of …scientific data and their associated algorithms
+and workflows.”[^1]  These principles are extremely powerful but as has been
+repeatedly noted, the routine implementation of FAIR principles is a
+significant challenge.
+ 
+Imaging datasets present a particular challenge for implementing FAIR. The
+datasets are large, multidimensional and complex. Perhaps most importantly it
+is probably unrealistic to suggest that a single metadata standard will handle
+the huge diversity of imaging experiments and datasets. In the best possible
+case, it is likely that there will be families of metadata standards or
+flexible APIs, each tuned and designed for accessing specific types of imaging
+metadata.
+ 
+OME has been working on the image data publication problem for many years.
+Our recent work on the [Image Data Resource (IDR)](https://idr.openmicroscopy.org/) is an example of an added
+value database that integrates imaging data from many biological imaging
+datasets and links gene and drug perturbations with cell phenotypes[^2]. IDR
+focuses on reference image datasets, i.e. those datasets that have high levels
+of biological and molecular annotations and have a strong likelihood of re-use
+by the scientific community.
+ 
+Our work on IDR has been well-received and the resource is growing in size and
+usage.  However, IDR doesn’t address more routine data publication, for
+example the datasets that are not reference images, but are associated with a
+scientific publication in the biological sciences. For example, our lab in
+Dundee has recently published a paper that explores the interaction of a
+single protein Bod1 and the Ndc80 complex, a protein complex that mediates the
+attachment of microtubules to connect to chromosomes during cell division[^3].
+How to publish the imaging data associated with this paper?
+ 
+As you might guess, we’ve used OMERO to publish and link these data.  We’ve
+used our institutional OMERO server, and used an institutional DOI as the
+definitive link to the data. The datasets associated with this paper were
+imported into OMERO as part of the analysis workflows and then were moved into
+a public OMERO group for publication. The data can be browsed, searched,
+viewed and downloaded. We believe we’ve made the datasets “AIR”—Accessible,
+Interoperable and Reusable. Making these datasets truly Findable will take
```

Comment:
Strange to have capital but related to "AIR" maybe a "-" in front will make it clearer